crates/re_viewport/src/viewport.rs

```diff
@@ -124,6 +124,9 @@ impl<'a, 'b> Viewport<'a, 'b> {
             blueprint.tree.clone()
         };
 
+        let shadow_tree = blueprint.build_tree_from_containers();
+        re_log::debug!("shadow_tree: {shadow_tree:?}");
```

Comment:
note: you can use `dbg!` while developing instead of "temporary" debug logging that you easily forget (you cannot merge a PR with `dbg!` in it)